instance/instance.go

```diff
@@ -25,6 +25,13 @@ var (
 	ErrIllegalDomain = errors.New("Domain name contains illegal characters")
 )
 
+var (
+	// fsLocalCache is the FS Cache used for a classical (non-
+	// distributed) filesystem. It is base on a simple in-memory cache.
+	// There is no fundamental theory behind the size of the cache.
+	fsLocalCache = vfs.NewLocalCache(50)
```

Comment:
I think you can be a lot more aggressive on caching for local cache. Maybe, 256 instead of 50